### DISCUSSION TITLE
Complete the expression

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1033] [Galaxy Trivia] [answered]
> There are only two ways to live your life. One is as ...

**Bonus:** Name the author.

**Comment by peterbaccaro** [comment]
The expression is "**There are only two ways to live your life. One is as though nothing is a miracle. The other is as though everything is a miracle**" and it is attributed to **Albert Einstein**.

**Comment by thuongtruong1009** [comment]
"There are only two ways to live your life. One is as though nothing is a miracle. The other is as though everything is a miracle." - Albert Einstein.

**Comment by Ben74x** [comment, marked as answer]
There are only two ways to live your life. One is as though nothing is a miracle. The other is as though everything is a miracle. 

Albert Einstein

**Comment by NomanGhiasy** [comment]
There are only two ways to live your life. One is as though nothing is a miracle. The other is as though everything is a miracle.

Albert Einstein!!
